examples/lang/runtime/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "runtime"
```

Comment:
I would maybe call this example `calling_the_runtime` or something like that or `calling_back`